docs/content/dagster-cloud/getting-started/getting-started-with-hybrid-deployment.mdx

```diff
@@ -60,17 +62,11 @@ Continue to the next step when finished.
 
 ## Step 3: Deploy your code
 
-Now that you've set up your deployment, you'll need to tell it the location of your Dagster code.
-
-A code location specifies a single Python package or file that defines your Dagster code. Each Dagster Cloud deployment can include code from one or more code locations. Refer to the [Adding a code location guide](/dagster-cloud/developing-testing/code-locations) for more info.
+There are two options for deploying your code. For most Hybrid deployments, with the exception of those using a local agent, you will need to create a Docker image containing your code and then add a code location to Dagster Cloud that references the image. Typically, this build and deploy is done through CI/CD. It is also possible to deploy a code location manually, and this manual process would be used with the local agent.
```

Comment:
- It's great that this mentions images, that was sorely missing before
- I don't think its necc. true that you only update manually when you're using the local agent